examples/dt/uni05epsilon/control-plane/service-values.yaml

```diff
@@ -67,8 +67,6 @@ data:
       cinder_store_password = {{ .ServicePassword }}
       cinder_store_project_name = service
       cinder_catalog_info = volumev3::internalURL
-      # TODO: Enable multipath when OSPRH-7393 is resolved
-      # cinder_use_multipath = true
```

Comment:
Wait, isn't it wrong? I think default of this value is `false`, hence I had originally added `cinder_use_multipath = true` here – shoudn't this patch just remove the `# `? (Same for zeta and eta)

Reply:
right, I somehow assumed that the default was true, but I'm wrong [1]!
Updating the patch!

[1] https://github.com/openstack/glance_store/blob/master/glance_store/_drivers/cinder/store.py#L386